src/diagnostics/default_diagnostics.jl

```diff
@@ -66,73 +67,158 @@ function default_diagnostics(model::ClimaLand.AbstractModel, start_date, outdir)
     return default_diagnostics(model, start_date; output_writer)
 end
 
-# Bucket
+"""
+    default_diagnostics(model::AbstractModel{FT}, start_date; output_writer, output_vars = :short, average_period = :monthly)
+
+For a general AbstractModel, we need a specification of output_vars to determine which diagnostics to output.
+
+The input `output_vars` can have 3 values:
+- `:long` - all diagnostics are output
+- `:short` - a short list of diagnostics is output
+- `_::Vector{String}` - a user-defined list of diagnostics is output
+
```

Comment:
The elements of the vector must coincide with the `short_name` diagnostics available for the model.

(separately - we should add a docs page explaining what diagnostics are available for each model?)

Reply:
I meant adding it to the docstring, sorry, that wasnt very clear

Reply:
Ah yes, I'll explain this in the docstring